package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "combobo",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
don't bump the version in this commit - I will handle that as part of the release